xc7/yosys/synth.tcl

```diff
@@ -15,5 +15,9 @@ opt_clean
 
 setundef -zero -params
 stat
-write_blif -attr -cname -param $::env(OUT_EBLIF)
+write_blif -attr -cname -param \
+ -true VCC VCC \
+ -false GND GND \
+ -undef VCC VCC \
```

Comment:
I'm unsure if undefined actually be mapped to VCC? Can you add a comment about why it is a good idea?

Reply:
Because that is what Xilinx's tools do?